frontend/components/modal/CreateGroupModal.tsx

```diff
@@ -0,0 +1,104 @@
+import { Trans, msg, t } from "@lingui/macro";
+import { useLingui } from "@lingui/react";
+import { useRef } from "react";
+import { toast } from "react-toastify";
+import { useSWRConfig } from "swr";
+
+import { InputField } from "@/components/form/InputField";
+import { Button } from "@/components/input/Button";
+import { Modal, ModalProps } from "@/components/modal/modal";
+import { postRequest } from "@/util/api";
+
+/**
+ * Modal for creating groups
+ */
+export const CreateGroupModal = ({
+	visible,
+	setVisible,
+	...props
+}: ModalProps) => {
+	const { mutate } = useSWRConfig();
+	const { _ } = useLingui();
+
+	const groupNameInput = useRef<HTMLInputElement>(null);
+	const groupDescriptionInput = useRef<HTMLInputElement>(null);
+
+	return (
+		<Modal
+			header={_(msg`Create new Group`)}
+			visible={visible}
+			setVisible={setVisible}
+			{...props}
+		>
+			<form className="space-y-5">
+				<div className="space-y-3">
+					<InputField
+						id="groupNameInputFieldId"
+						type="text"
+						label={_(msg`Group Name`)}
+						inputFieldLabelStyle="text-gray-400"
+						required
+						placeholder={_(msg`Enter group name`)}
+						className="bg-gray-100 px-2 py-3"
+						inputFieldSize="5xs"
+						autoFocus
+						ref={groupNameInput}
+					></InputField>
+					<InputField
+						id="groupDescriptionInputFieldId"
+						type="text"
+						label={_(msg`Group description`)}
```

Comment:
```suggestion
						label={_(msg`Group Description`)}
```
I think we should use Title Case here, as we do with the group name as well.

Reply:
fixed in a39d1014ca5d856c1da2ee500c221e7f3503f244